clinic/clinic-introduction.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Clinic Diagnostic Service Overview
+summary: Introduces the Clinic diagnostic service, including tool components, usage scenarios, and implementation principles.
+---
+
+## Clinic Diagnostic Service Overview
+
+Clinic is a diagnostic service provided by PingCAP for TiDB clusters. To the clusters deployed using TiUP and TiDB Operator, this service helps to troubleshoot cluster problems remotely and perform a quick check of the cluster status locally. With Clinic diagnostic service, you can ensure the stable operation of your TiDB cluster for its full life-cycle, predict potential issues, reduce the probability of the issues, troubleshoot problems quickly, and fix the problems.
+
+The Clinic diagnostic service is currently in the Beta testing stage, so only the invited users can use the service. The service provides the following two components to diagnose clusters:
+
+- Diag: The tool deployed on the cluster side. Diag is used to collect cluster diagnostic data, upload the diagnostic data to Clinic Server, and perform a quick health check locally on the cluster. For the list of diagnostic data collected by Diag, see [Clinic Diagnostic Data](/clinic/clinic-data-instruction-for-tiup.md).
+
+    > **Note:**
+    >
+    > - Diag temporarily **does not support** collecting data in the clusters with TLS encryption enabled and the clusters deployed using TiDB Ansible.
+    > - The Clinic diagnostic service is currently in the Beta testing stage, so only the invited users can use the service. If you need to upload data to Clinic Server using Diag, you should get a trial account from the PingCAP technical support staff you contacted before.
+
+- Clinic Server: The cloud service deployed in the cloud. By providing diagnostic services in SaaS mode, Clinic Server can not only receive the diagnostic data uploaded to the Clinic Server, but also store the data, view and diagnose the uploaded data online, and provide cluster diagnostic reports.
```

Comment:
```suggestion
- Clinic Server: The cloud service deployed in the cloud. By providing diagnostic services in SaaS mode, Clinic Server can not only receive the diagnostic data uploaded to the Clinic Server, but also store the data. It can provide online diagnostic environment for the uploaded data , and cluster diagnostic reports.
```

Reply:
server 自己不能 view and diagnose the uploaded data online ，是给技术人员提供  online diagnostic environment